encointer/personhood-oracle
# merge upstream multi parentchain feature

get all changes from https://github.com/integritee-network/worker/pull/1420

test instructions:
1. put `integritee-node` and `encointer-node-notee` into `./bin` directory 
2. `SGX_MODE=SW WORKER_FEATURES=dcap WORKER_MODE=teeracle make`
3. `./local-setup/launch.py ./local-setup/config/pesonhood-oracle.json` will bring up integritee, encointer and the personhood oracle service
4. `./integritee-cli -p 9966 personhood-oracle issue-nostr-badge //Alice npub1xq33nsus0d2d00jzdea4unl08p35cd05md7mmgtxky5sncsjgxvqw2p77y sqm1v79dF6b wss://relay.damus.io`
5. you should see a success message
